.circleci/config.yml

```diff
@@ -46,7 +45,6 @@ jobs:
           command: |
             export CONTEXT="production"
             yarn install
-            chmod +x deploy.sh
             ./deploy.sh
```

Comment:
I think this command will no longer work if the file is not marked as executable? It wil need `sh deploy.sh` if it's necessary to not mark this as executable

Reply:
Ignore my comment, the deploy.sh file is already committed with the executable flag :)